.github/workflows/ci-run-test.yml

```diff
@@ -140,6 +140,10 @@ jobs:
           } >> "$GITHUB_ENV"
           # For debugging:
           ps aux
+      - name: Install tsc on windows and macos
```

Comment:
This is needed to run the new integration test.  I think it's not been needed so far because we're running most integration tests only on linux.